src/MonthView/phone/CalendarMonth/CalendarMonth.tsx

```diff
@@ -0,0 +1,200 @@
+import moment from "moment"
+import * as React from "react"
+import { LayoutChangeEvent, View } from "react-native"
+import { DateNumber, Month, Task, TaskSchedule, User } from "../../../shared/model"
+import { styles } from "./CalendarMonth.styles"
+import { DateCalendarBox } from "./DateCalendarBox"
+import { MonthSelection } from "./MonthSelection"
+import { WeekDayHeader } from "./WeekDayHeader"
+
+export interface TaskAndUser {
+    user: User,
+    tasks: Task[]
+}
+
+interface CalendarMonthProps {
+    data?: TaskAndUser[]
+}
+
+interface CalendarMonthState {
+    dateBoxWidth: number,
+    selectedDate?: DateNumber,
+    currentMonth: Month,
+    currentYear: number,
+}
+
+interface BoxToDateNumberMap {
+    [key: number]: DateObject
+}
+
+interface ScheduleToUserMap {
+    [key: number]: User[]
+}
+
+interface DateObject {
+    dateNumber: DateNumber
+    isWeekend: boolean
+    usersBusy: User[]
+    isSelected: boolean
+}
+
+const nBoxToDateMapper = (year: number, month: number,
```

Comment:
Is it just me or the formatting here looks a little weird, like each argument on a new line?